DAS/das/V2_mqtt_playback.py

```diff
@@ -0,0 +1,120 @@
+import sys
```

Comment:
Sorry, just realised another thing - this is meant to replace `mqtt_test.py` isn't it? Right now it only duplicates it.
Remove `mqtt_test.py` and copy over the new `mhp` topic usages into here. Then good to go, promise!

Reply:
Should be fixed now!